google/cloud/speech/doc/main.dox

```diff
@@ -42,6 +42,10 @@ which should give you a taste of the Cloud Speech-to-Text API C++ client library
 
 <!-- inject-endpoint-env-vars-start -->
 
+- `GOOGLE_CLOUD_CPP_SPEECH_ENDPOINT=...` overrides the
```

Comment:
Seems like some sort of `sort -u` would be useful in the script that updates these things?

Reply:
I lean more towards including the namespace, such as: `speech_v2::MakeSpeechConnection()`